from_cpython/Modules/getpath.c

```diff
@@ -591,15 +592,20 @@ calculate_path(void)
     bufsz += strlen(zip_path) + 1;
     bufsz += strlen(exec_prefix) + 1;
 
-    // Pyston change: add from_cpython/Lib and lib_pyston
+    // Pyston change: add from_cpython/Lib, lib_pyston and Lib/lib-tk
```

Comment:
I'm pretty skeptical of this change -- we shouldn't have to add any libraries here.  Can we just see how CPython ends up not needing this, and try to copy it?